main.go

```diff
@@ -121,39 +119,38 @@ func updateGame() {
 		}
 	}
 
-	if frames%wallMovementInterval == 0 {
-		for _, wall := range walls {
+	for _, wall := range walls {
+		wall.move()
+		if frames%4 == 0 {
 			wall.move()
 		}
```

Comment:
It used to shift the wall by 8 pixels every 10 frames. It gives the ratio of 4 to 5. Which means that, to preserve the original wall speed, we need to shift the wall one more pixel every 4 frames.